test/check-machines-create

```diff
@@ -1267,7 +1267,7 @@ class TestMachinesCreate(VirtualMachinesCase):
                 # In this case don't use runuser -l admin because we get errors 'authentication unavailable'
                 listCmd = "virsh -c qemu:///system dumpxml --inactive {0}".format(name)
 
-            wait(lambda: "6" in self.machine.execute(listCmd + " | grep cockpit_machines | wc -l"))
+            wait(lambda: "7" in self.machine.execute(listCmd + " | grep cockpit_machines | wc -l"))
```

Comment:
Why would the running counter change here? I don't see any changes to the test which would start a new machine?

Reply:
So this I explained it in the commit message. I don't get why, by building now the XML from python, I don't seem to have control to which element has the xmlns tags.

Now it looks like this:
```
  <metadata xmlns:libosinfo="http://libosinfo.org/xmlns/libvirt/domain/1.0" xmlns:cockpit_machines="https://github.com/cockpit-project/cockpit-machines">
    <libosinfo:libosinfo>
      <libosinfo:os id="http://centos.org/centos-stream/8"/>
    </libosinfo:libosinfo>
    <cockpit_machines:data>
      <cockpit_machines:has_install_phase>false</cockpit_machines:has_install_phase>
      <cockpit_machines:install_source_type>os</cockpit_machines:install_source_type>
      <cockpit_machines:install_source/>
      <cockpit_machines:os_variant>centos-stream8</cockpit_machines:os_variant>
    </cockpit_machines:data>
  </metadata>
```

Previously the xmlns:libosinfo="http://libosinfo.org/xmlns/libvirt/domain/1.0" and xmlns:cockpit_machines="https://github.com/cockpit-project/cockpit-machines" tags where part of the libosinfo and cockpit_macines:data elements respectively

I thought this is some XML standard that ElementTree has, to add the xmlnfs to the parents.

Reply:
Ooh! I was confused. I thought this was `virsh list`, but it is `virsh dumpxml`. Then the numerical change makes sense. (It's a bit odd to expect number of lines instead of doing a substring match for the actual things you expect, but that's follow-up matter). Maybe rename `listCmd` to `dumpCmd`?

Reply:
FTR: The previous structure was a bug -- you can't declare a namespace *inside* of a tag that already uses it. So what ET produces here is correct, and this is a bug fix. 

Still keeping this thread open for the confusing `listCmd` variable

Reply:
FIxed.